package.json

```diff
@@ -11,19 +18,41 @@
   "author": "Lenoir Zamboni",
   "license": "MIT",
   "devDependencies": {
-    "react-native": "0.41.2"
+    "jest": "^26.1.0",
+    "react-native": "0.61.2"
   },
   "peerDependencies": {
-    "react-native": "0.41.2"
+    "react-native": "0.61.2"
+  },
+  "scripts": {
+    "test": "jest --coverage"
+  },
+  "jest": {
+    "automock": false,
+    "preset": "react-native",
+    "moduleFileExtensions": [
+      "js"
+    ],
+    "transform": {
+      "^.+\\.js$": "<rootDir>/node_modules/react-native/jest/preprocessor.js"
+    }
   },
   "nativePackage": true,
   "homepage": "https://github.com/loadsmart/rn-salesforce-chat#readme",
-  "dependencies": {},
+  "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "husky": "^4.2.5"
+  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/loadsmart/rn-salesforce-chat.git"
   },
   "bugs": {
     "url": "https://github.com/loadsmart/rn-salesforce-chat/issues"
+  },
+  "husky": {
+    "hooks": {
+      "pre-push": "npm test"
```

Comment:
@lenoirzamboni How long is the test execution taking today?

Reply:
@barbosa About five seconds. Why?

Reply:
Just because if this was taking too long, and giving that we should be always running tests for the changes we do, this pre push hook would just run it one more time. But 5 seconds is fast enough 👍